client/package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "yarn lint:format && yarn lint:fix "
   },
   "dependencies": {
-    "@tanstack/react-query": "^4.19.1",
+    "@tanstack/react-query": "^4.20.9",
```

Comment:
Shouldn't there be @ant-design/icons in dependencies?

Reply:
Well, I don't see them on the master either

Reply:
Hmm, was it working for you right away or did you have to install the package?

Reply:
I think I did yarn command for installation and that's all